### DISCUSSION TITLE
details html tag support

### DISCUSSION THREAD

**Discussion by nosovk** [pngwn/MDsveX#272] [Q&A] [answered]
Currently there is an html tag called `details`

```html
<details>
   <summary>Some heading</summary>
# h1 text
[link](link)
</details>
```

Is there some way to proceed contents of details tag?
Currently this block parsed as html and being inserted as is into resulting html.

**Comment by pngwn** [comment, marked as answer]
Add new lines around anything you want to be parsed and transformed:

```markdown
<details>
   <summary>Some heading</summary>

# h1 text
[link](link)

</details>
```

**Comment by AyushSehrawat** [comment]
@pngwn when trying this now on svelte 4 with sveltekit 2, i get the error that `</details> attempted to close an element that was not open`
